Pie (#4953)
Codecademy/docs: content/plotly/concepts/express/terms/pie/pie.md

> # Plotly Pie Charts
> 
> ## .pie()

Reviewer: ```suggestion
---
Title: '.pie()'
Description: 'Creates a pie chart, a circular statistical graphic divided into slices to illustrate numerical proportions.'
Subjects:
  - 'Data Science'
  - 'Data Visualization'
Tags:
  - 'Graphs'
  - 'Libraries'
  - 'Plotly'
  - 'Data'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

Reviewer: Meta data is needed at the start of every entry

> In python, we can use *.pie()* from the *plotly library* to display data in the form of a pie chart.
> This is such a simple tool to use and can transform data into visually apealing ways for readers
> to get a better idea about the relationships between variables.

Reviewer: ```suggestion
In Python, the **`pie()`** method from the Plotly library is used to create a pie chart, which visually represents data proportions. This method helps transform data into an appealing visual format, making it easier for readers to understand the relationships between variables.
```

Reviewer: Reframed the sentence for better clarity. 
- Avoid using "we", "you" in the sentences.
- The first instance of the term should be in bold and backticks: **`.pie()`**

> 

Reviewer: After the introduction paragraph, use the `## Syntax` block to provide the syntax of the `.pie()` method. Additionally, describe the parameters in the syntax in the following format:
- `parameter1`: Its description
- `parameter2`: its description
- `parametern`: its description

> ## Example
> ```

Reviewer: ```suggestion
## Example

The example below shows the usage of the `.pie()` method as follows:

```py
```

import plotly.express as px
import numpy

# Example data
data = [45, 55, 30, 15]
names = ['A', 'B', 'C', 'D']

# Create a plot
fig = px.pie(values=data, names=names)

# Display the plot
fig.show() 
```

1. First, you need to make sure that you have the relevant modules imported:
```
import plotly.express as px
```
Here we import the *plotly* library and to give it the alias px.

2. We of course then, need to define our data that we want to include in the pie chart as well as the names to match:
```
data = [45, 55, 30, 15]
names = ['A', 'B', 'C', 'D']
```

3. Next, we need to use .pie() on px and pass our data as the arguement for the values and names as the value for names. We create a variable called fig to hold this for later:
```
fig = px.pie(values=data, names=names)
```

4. Finally, we need to show our data visualization, we can do this by using .show() on the fig variable we created earlier:
```
fig.show()

> ```

Reviewer: These explanations can be added in the comments of the code itself
